README.md

```diff
@@ -19,6 +19,7 @@ You can also find information on how to contribute to this repository, either by
   - [Song scripts](#song-scripts)
   - [Project scripts](#project-scripts)
 - [Getting help](#getting-help)
+- [History](#history)
```

Comment:
Would prefer it to be `Background` instead of `History`

Reply:
Resolved with fixup commit.